llvm/include/llvm/DebugInfo/DWARF/DWARFDebugLine.h

```diff
@@ -78,7 +78,7 @@ class DWARFDebugLine {
     /// The maximum number of individual operations that may be encoded in an
     /// instruction.
     uint8_t MaxOpsPerInst;
-    /// The initial value of theis_stmtregister.
+    /// The initial value of the is_stmt register.
```

Comment:
Could you commit this unrelated change separately (doesn't need precommit review/a PR, can be pushed directly if you have commit access)